src/pages/BasePersonalisationPage.scss

```diff
@@ -112,4 +112,24 @@
         text-align: center;
     }
 
+    .hasFloatingDoneButton {
+        padding-bottom: 80px;
+
+        .done-button {
+            @include riseIn(1s, 0s, forwards);
+
+            position: fixed;
```

Comment:
I think it miiight be possible to do what I want using flex box but I can't find a good example